components/PlayerArea.jsx

```diff
@@ -17,19 +17,19 @@ function PlayerArea({ onTap, player }) {
         height: 50,
         backgroundColor: playerColors[player]
       }}
+      onTouchStart={() => {
+        onTap(player);
+        setTapCount(tapCount + 1);
+      }}
     >
       <View
         style={{
           height: "100%",
           //   justifyContent: "center",
           alignItems: "center"
         }}
-        onTouchStart={() => {
-          onTap(player);
-          setTapCount(tapCount + 1);
-        }}
       >
-        {player === 1 ? <View style={{ flex: 7 }} /> : <></>}
+        {player === 'playerTwo' ? <View style={{ flex: 7 }} /> : <></>}
```

Comment:
Let's not use raw strings for the players themselves because it makes for easy typos.

Can you define the variables at the top of the file or something and then use them as variables?

```
PLAYER_TWO = 'playerTwo';
// some code
player === PLAYER_TWO
```

Reply:
How do I make this more DRY across the code, I feel like I'm defining these all over the place. Can I just define these somewhere and import them or is there a better way of doing this?